dat/ships/drone_heavy.xml

```diff
@@ -24,8 +24,8 @@
  <slots>
   <weapon size="medium" x="8" y="1" h="-5" />
   <weapon size="medium" x="8" y="-1" h="-5" />
-  <weapon size="small" x="5" y="5" h="-5" />
-  <weapon size="small" x="5" y="-5" h="-5" />
+  <weapon size="medium" x="5" y="5" h="-5" />
```

Comment:
It's a bit weird that the heavy drone has all medium weapon slots while being a small ship. Furthermore, if I'm correct, this will cause it to only have the medium shatterer missiles and not have heavy neutron blasters. It might be better to leave this as is (2x and 2x small).